src/lib/coapcore/Cargo.toml

```diff
@@ -26,9 +26,9 @@ coap-message-utils = "0.3.3"
 coap-numbers = "0.2.3"
```

Comment:
(again as a conversation so we don't forget) Would you revert https://github.com/future-proof-iot/RIOT-rs/commit/d351ee888ae97ddf5928d92778f0ec5f8243e3dc here (as soon as that landed in main)?